common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -3740,6 +3741,15 @@ object CelebornConf extends Logging {
       .timeConf(TimeUnit.MILLISECONDS)
       .createWithDefaultString("10s")
 
+  val WORKER_RESUME_BY_PINNED_MEMORY_KEEP_TIME: ConfigEntry[Long] =
```

Comment:
How should the user measure this configuration and configure it appropriately? What facts help users adjust this configuration value?